src/Login/Login.js

```diff
@@ -72,12 +72,11 @@ class Login extends Component {
               onChange={this.updateText}
             />
           </p>
-          <button 
-            data-testid="login-button" onClick={this.login} type="button">Log In</button>
+          <button data-testid="login-button" onClick={this.login} type="button">Log In</button>
+          {this.state.error &&
+            <p id="error-message">Invalid Username/Password</p>
+          }
```

Comment:
Nice error handling!

Reply:
Additionally, move all of your fetches into an api helpers file. This will make your components easier to test & clean up your code